build/elasticsearch/bin/es-docker

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Run Elasticsearch and allow setting default settings via env vars
+#
+# e.g. Setting the env var -Ecluster.name=testcluster
+#
+# will cause Elasticsearch to be invoked with -Ecluster.name=testcluster
+#
+# see https://www.elastic.co/guide/en/elasticsearch/reference/5.0/settings.html#_setting_default_settings
+
+es_opts=''
+
+while IFS='='  read -r envvar_key envvar_value
+do
+    case $envvar_key in
+    [a-z.]*)
```

Comment:
As discussed, switched to the use of regexp and (hopefully) fixed by https://github.com/elastic/elasticsearch-docker/commit/6957c9f863a4bb6563e0883966c5e3cc5344f1c3